examples/demo8.py

```diff
@@ -144,28 +141,17 @@ def main(argv):
     # save the catalog (or whatever) in the config dict in the way that further processing
     # functions expect.  However, we don't have any input field, so we don't need it here.
     #
-    # To build the files, the Process function then calls one of the following:
+    # To build the files, the Process function then calls the following to build each file:
     #
-    #     galsim.config.BuildFits(file_name, config)               -- build a regular fits file
-    #     galsim.config.BuildMultiFits(file_name, nimages, config) -- build a multi-extension fits 
-    #                                                                 file
-    #     galsim.config.BuildDataCube(file_name, nimages, config)  -- build a fits data cube
+    #     galsim.config.BuildFile(config, file_num)
```

Comment:
Now that there's just single element here, it's probably best to rephrase this sentence.